ArmPkg/ArmPkg.dsc

```diff
@@ -161,7 +175,11 @@
   ArmPkg/Universal/Smbios/ProcessorSubClassDxe/ProcessorSubClassDxe.inf
   ArmPkg/Universal/Smbios/SmbiosMiscDxe/SmbiosMiscDxe.inf
   ArmPkg/Universal/Smbios/OemMiscLibNull/OemMiscLibNull.inf
-
+  # MU_CHANGE [BEGIN]
+  ArmPkg/Drivers/StandaloneMmCpu/StandaloneMmCpu.inf
+  ArmPkg/Library/StandaloneMmCoreEntryPoint/StandaloneMmCoreEntryPoint.inf
+  ArmPkg/Drivers/SmmVariablePei/SmmVariablePei.inf

```

Comment:
Please hold off on this module. I thought I upstreamed this one to edk2 already.

Reply:
Yup, it should be replaced by this: https://github.com/tianocore/edk2/blob/master/MdeModulePkg/Universal/Variable/MmVariablePei.

Can we drop anything related to `SmmVariablePei.inf` in this PR?

Reply:
Thanks. It moving to a different package/name didn't get found in search. Dropped it.